lib/ReactViews/Mobile/mobile-header.scss

```diff
@@ -60,6 +61,8 @@
   padding-right: $padding;
   position: relative;
   width: 40px;
+  border-radius: 4px;
+
```

Comment:
This is a repetition, I think we should fix it when we remove `scss` and move it to styled-components.